src/index.js

```diff
@@ -0,0 +1,14 @@
+const renderLobby = require('./views/lobby');
+const {attemptToRenderTable} = require('./views/table');
+// import DeckrTable from './phaser'
```

Comment:
Unless you're planning on moving this back in at some point you can probably delete it